src/components/Doc/ComponentDoc.js

```diff
@@ -79,6 +87,38 @@ export const ComponentDoc = ({
         )}
       </Box>
 
+      {designSystemLink && (
+        <ThemeContext.Extend
```

Comment:
Why do we need to override the theme? Can we use the default?

Reply:
I think it was because I couldn't find a way to make the default notification visually match what we agreed on, and it was easier to override the theme and manually specify certain parameters.

Reply:
I think the main thing we wanted was to leverage Notification component for layout and text styling, but needed to add background color/rounding/direction that differs from grommet's theme.

https://storybook.grommet.io/?path=/story/visualizations-notification-status--status

Reply:
Alternately, Jeffrey had explored just building the entire thing with Box, but getting to leverage Notification felt nice if possible (but does require theme overrides since these things aren't adjustable via props) @ericsoderberghp

Reply:
Maybe the default grommet theming looks okay though and we just stick with that?

Reply:
Semantically, this feels like it should be a Notification, something to call attention to. If the grommet theme isn't working, then perhaps it means we should change the grommet theme. I'm inclined to use the components we've got as much as possible and only tweak them when we need to use them in distinct ways. All that said, I'll leave it up to the two of you to decide how you want to proceed. I'm OK taking this as is and submitting a new issue to re-evaluate the grommet theme for Notification, if preferred.

Reply:
I chatted with Aditi and she didn't have any allergic reactions to the grommet default styling. In the spirit of keeping things lean, let's go with the default grommet styling for this first pass.

Next steps after merging -->

We can touch base with some of the developers I interviewed and open-ended let them know we made some small changes to the docs pages and let them browse. If they notice the new link, awesome! If no one notices it, that will indicate visibility is too low and we might need to rethink the visual treatment.